indigo/indigo/src/main/scala/indigo/platform/renderer/webgl1/RendererWebGL1.scala

```diff
@@ -1,16 +1,26 @@
 package indigo.platform.renderer.webgl1
 
+import indigo.AssetName
+import indigo.AssetPath
 import indigo.platform.assets.AtlasId
```

Comment:
I'd be tempted to drop this from the WebGL1 implementation. WebGL1 is very limited - really only good enough to display a message saying "don't use this" - and I'd hope no-one is trying to build anything screen capture worthy in it. What do you think?

Reply:
I think all the time we have a WebGL1 implementation we should support it.. It doesn't add much work in this case, so I think we should keep it